### PR TITLE
Feat/projects page

### PR DESCRIPTION
**Pull request by AlanSyue**
# Description
Create a new projects page.

## Followed up
- Need to discuss with the `[mentor information component](https://app.asana.com/0/1200963218344861/1201889627075445/f)`
- Need to discuss how to [redirect the page using anchor when clicking the next page component](https://app.asana.com/0/1200963218344861/1201889716532291/f).

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] You can test it in the development environment. [link](https://stage.itseed.tw/projects)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: components/templates/Projects/CareerEvents/CareerEvents.module.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    margin-top: 40px;
+
+    > h2 {
+            display: flex;
+            align-items: center;
+    }
+}
+
+.circle {
+    float: left;
```
Comment (Nick0603) [inline components/templates/Projects/CareerEvents/CareerEvents.module.scss:26]: I think the style attribute is redundant.

File: components/templates/Projects/CareerEvents/CareerEvents.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const CareerEvents = () => {
+  return (
+    <>
+      <div>
+        <SectionWrapper>
+          <div className={styles.topicSection}>
+            <h2>職涯活動</h2>
+            <p>
+              職涯活動：由第 18
+              屆職涯召部團隊也舉辦了一系列的職涯活動，以四大核心價值「分享、回饋、回顧、前瞻」，來協助學員在不同的產業與職務能有更深入的了解，盡情探索各種未來方向，並成為更具有競爭力的市場人才，以此籌畫了各類型職涯活動！
```
Comment (Nick0603) [inline components/templates/Projects/CareerEvents/CareerEvents.tsx:14]: Why does this split line here?

File: components/templates/Projects/index.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        type={BannerType.general}
+        primaryTitle="四大實作專案"
+        subTitle=""
+        description="資訊種子培訓計畫不同於大學一貫的授課方式，讓學員在實踐中學習。透過執行 4 大專案，參與 10+ 堂來自業界講師的課程，了解業界生態，並探索自己未來的職涯方向。
+培養跨領域合作、解決問題的思維等職場必備的能力，成為能踏入職場的人才。"
+      />
+      <TabNav
+        tabs={tabs}
+        activeTab={project}
+        onTabClick={(tab: ProjectName) => {
```
Comment (Nick0603) [inline components/templates/Projects/index.tsx:65]: The activeTab style is not working now.

I think it is caused by the detection mechanism that I use `tav.activeTab` to match `tab.text`.

You can solve it by
1. Use the tab text to the anchor name

2. Extend the TabNav component to know how to match key for active tab.

File: components/templates/Projects/index.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        onStickyChange={layoutContext.setIsSubNavStuck}
+      />
+      {(() => {
+        switch (project) {
+          case Project.tuv:
+            return (
+              <>
+                <GoalSection projectDescription={Goals[project].projectDescription} />
+                <AchievementSection />
+                <NextSection title="職涯專案" path="/projects#career" />
```
Comment (Nick0603) [inline components/templates/Projects/index.tsx:81]: The page no handle for hash change event, so it's nothing to happed when user click NextSection component.

Solutions:
1. Add onClick property to  NextSection component, and let the `NextSection` prefer to use onClick than using `path` to change.

2. Add a `hash change event` for the page, and don't change `set

**Review by Nick0603 — CHANGES_REQUESTED** [review]
The path for Navigation ( or named by Header) is not updated to new path `/projects` with the feature.

File: components/templates/Projects/CareerEvents/CareerEvents.tsx (hunk 2 above)
Comment (AlanSyue) [reply components/templates/Projects/CareerEvents/CareerEvents.tsx:14]: It is split by the `yarn format:write`

**Review by Nick0603 — APPROVED** [review]
LGTM

**Pull request merged**